### PR TITLE
feat: style transfer(android)

### PR DESCRIPTION
**Pull request by NorbertKlockiewicz**
## Description
This pr introduces implementation of style transfer with openCV for android

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (improves or adds clarity to existing documentation)

### Tested on
- [ ] iOS
- [x] Android

### Checklist
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly
- [ ] My changes generate no new warnings

### CODE REVIEW

File: android/src/main/java/com/swmansion/rnexecutorch/utils/ETError.kt [hunk 1]
```diff
@@ -2,28 +2,29 @@ package com.swmansion.rnexecutorch.utils
 
 enum class ETError(val code: Int) {
     InvalidModelPath(0xff),
-    
-    // System errors
+    FileWriteFailed(0x67),
```
Comment (mkopcins) [inline android/src/main/java/com/swmansion/rnexecutorch/utils/ETError.kt:6]: why this value?

Comment (NorbertKlockiewicz) [reply android/src/main/java/com/swmansion/rnexecutorch/utils/ETError.kt:6]: It's based on errors defined in typescript, errors that are not from executorch are starting from 100.

**Comment by NorbertKlockiewicz** [comment]
I was debugging problem with release version and cache was the problem, basically it should work without any changes required as I cloned new repo and it's working fine

File: android/src/main/java/com/swmansion/rnexecutorch/utils/ETError.kt (hunk 1 above)
Comment (mkopcins) [reply android/src/main/java/com/swmansion/rnexecutorch/utils/ETError.kt:6]: iirc you formatted this differently in ios, do the same here

**Review by mkopcins — APPROVED** [review]

**Pull request merged**